src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -55,12 +56,13 @@ public void intakeIn(boolean isReverse) {
     public void intakePosition() {
         //NOTE: negative constant speed for down, positive for up
         if (RobotContainer.kOperateRobot) {
+            moving = true;
             do {
-                if(!kintakeCallibrated){
+                if(!kIntakeCallibrated){
```

Comment:
You will always hit `!kIntakeCallibrated`, add && `!upLimitSwitch.get()` to skip if it's already up.